Session/Conversations/ConversationVC.swift

```diff
@@ -341,7 +361,8 @@ final class ConversationVC: BaseVC, ConversationSearchControllerDelegate, UITabl
         // nav will be offset incorrectly during the push animation (unfortunately the profile icon still
         // doesn't appear until after the animation, I assume it's taking a snapshot or something, but
         // there isn't much we can do about that unfortunately)
-        updateNavBarButtons(threadData: nil, initialVariant: self.viewModel.initialThreadVariant)
+        let isNoteToSelf: Bool = self.viewModel.threadData.threadId == getUserHexEncodedPublicKey()
```

Comment:
When this code runs, `self.viewModel.threadData.threadIsNoteToSelf` is always `nil`. So I'll need to give the value to `threadIsNoteToSelf` either in the `init` method of `ConversationVC` or in the `SessionThreadViewModel` making `threadIsNoteToSelf` a variable to return `threadId == getUserHexEncodedPublicKey()` rather than give it value within the sql query. I guess the latter is preferred?

Reply:
Ah right 😞, yea I think updating the initialisation of the `threadData` in the `ConversationViewModel` to have `threadIsNoteToSelf: (self.threadId == getUserHexEncodedPublicKey()),` (line ~106) would be a bit cleaner since it means the logic stays in the view model